dev-packages/e2e-tests/test-applications/create-remix-app-express-vite-dev/tests/behaviour-server.test.ts

```diff
@@ -33,18 +33,21 @@ test('Sends two linked transactions (server & client) to Sentry', async ({ page
 
   const httpServerTraceId = httpServerTransaction.contexts?.trace?.trace_id;
   const httpServerSpanId = httpServerTransaction.contexts?.trace?.span_id;
+  const loaderSpanId = httpServerTransaction.spans.find(
```

Comment:
Is the loader span connected to the http.server span?

Reply:
Yes, I added a test to validate now. https://github.com/getsentry/sentry-javascript/pull/12110/commits/372e8be4a11343cc09ac7923baed789214b839c0